src/pyDE1/scale/felicita_arc.py

```diff
@@ -0,0 +1,147 @@
+"""
+Copyright © 2022 Mimoja. All Rights Reserved.
+
+License for this software, part of the pyDE1 package, is granted under
+GNU General Public License v3.0 only
+SPDX-License-Identifier: GPL-3.0-only
+"""
+
+import enum
+import sys
+import time
+from typing import Callable
+
+import pyDE1
+from pyDE1.scale import Scale
+from pyDE1.scale.events import ScaleWeightUpdate
+
+logger = pyDE1.getLogger('Scale.FelicitaArc')
+
+class FelicitaArc(Scale):
+
+    def __init__(self):
+        super(FelicitaArc, self).__init__()
+        self._nominal_period = 0.1  # seconds per sample
+        self._minimum_tare_request_interval = 2.5 * self._nominal_period # Guessed, not tested
+        self._sensor_lag = 0.45  # seconds, including all transit delays
+        self._tare_timeout = 1.0  # seconds until considered coincidence
+        self._tare_threshold = 0.05  # grams, within this, considered "at zero"
+
+        # Enable tare on button 1, hold UUID if need to unsubscribe
+        self._button_1_tare_subscriber_id = None
+        self._supervisor_button = None
+
+        # Linux, at least on an RPi 3B, needs response=True for write_gatt_char
+        self._write_gatt_char_response = sys.platform == 'linux'
+
+    async def standard_initialization(self, hold_notification=False):
+        await super(FelicitaArc, self).standard_initialization(
+            hold_notification=True)
+        if not hold_notification:
+            await self._notify_ready()
+
+    async def update_self_from_device(self):
```

Comment:
Might as well drop this. It isn't an expected method of the parent class.

On the Acaia Lunar 2021, it seems to report details of the Bluetooth module, not the scale itself. so not all that useful
```
    # These seem to be from https://www.microchip.com/en-us/product/BM70
    # rather than related to the Lunar (running v1.00.001 FW at the time)
    SYSTEM_ID       = '2a23'    # b'\x00\x00\x00\x00\x00\x00\x00\x00'
    MODEL_NUMBER    = '2a24'    # b'BM70'
    SERIAL_NUMBER   = '2a25'    # b'0000'
    FIRMWARE_REV    = '2a26'    # b'009010'
    HARDWARE_REV    = '2a27'    # b'5505 102_LESDK'
    SOFTWARE_REV    = '2a28'    # b'0000'
    MANUFACTURER    = '2a29'    # b'MCHP'
    # This is apparently related to medical devices, like glucose meters
    IEEE_11073_20601 = '2a2a'   # b'\x00\x00\x00\x00\x01\x00\x00\x00'
```